pom.xml

```diff
@@ -171,7 +171,7 @@
     <ethereum-core.version>1.12.0-v0.5.0</ethereum-core.version>
     <grpc.version>1.39.0</grpc.version>
     <guava.version>31.0.1-jre</guava.version>
-    <hapi-proto.version>0.20.0-SNAPSHOT</hapi-proto.version>
+    <hapi-proto.version>0.21.0-ecdsa-SNAPSHOT</hapi-proto.version>
```

Comment:
We can use `0.21.0-SNAPSHOT` now I believe.

Reply:
Agreed. Modified it. Thanks Michael !